tests/conformance_upb.c

```diff
@@ -0,0 +1,162 @@
+/* This is a upb implementation of the upb conformance tests, see:
```

Comment:
As of this PR it doesn't work at all, so I didn't add it to the Makefile until the next (in-progress) CL.